### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter month about July 2020

### PR DESCRIPTION
**Pull request by Echolon**
Created the draft for the next release August 2020.

Everyone participating, please just don't drop links - rather try to make a short and decent formulated description what is the main outcome of your source. You don't need to write formal, be passionately why this motivates you or what you work on! 

You can make a simple comment or create a merge request to make things easy going for review. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

You don't need a command-line tool or any knowledge of git to commit your changes. (Of course, pull requests are highly appreciated.)

- If you have write access add your comment as follows: Click the `Files changed` ribbon, then the three dots on the top right, then `Edit file`. Finally add you comment.
- Just paste your prepared text here below or the our channel (commteam@muc.xmpp.org)

This is our planed timeline for this draft:

XMPP Newsletter Timeline

CC: @jcbrand @Nyco @SeveFP

**Note: If you create a merge request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

Status:
- [X] Links collection
- [X] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

**Comment by Echolon** [comment]
Reminder: Move first sections to the end

**Comment by jerome-poisson** [comment]
@pmaziere Hi, thank you for advertising SàT website moving on SàT itself, but this news is one year old (I probably should update official news more often, my blog is currently a better way to stay up to date: https://www.goffi.org/).
I should have some news before the end of the month though :)

**Comment by pmaziere** [comment]
> @pmaziere Hi, thank you for advertising SàT website moving on SàT itself, but this news is one year old

my bad ! I did not check the year…  :(
@Echolon, I guess this should be revert then !

**Comment by Echolon** [comment]
@kriztan added as you suggested dddddfb

**Comment by eeeeeta** [comment]
News item from me (can PR if you'd prefer):

[whatsxmpp](https://git.theta.eu.org/eta/whatsxmpp), the WhatsApp (Web) to XMPP bridge, has seen a variety of bugfixes and improvements this month, mainly as a result of various users running the bridge and running into errors. We also switched to using the Nix package manager to build our Docker images, resulting in more reproducible, faster builds!
We're getting closer and closer to a stable 0.1 release, and the basic functionality is already working and battle-tested -- only a few more stability improvements and missing features (such as proper MUC history support and file uploading via native WhatsApp servers) are missing! Please do pop by our MUC [whatsxmpp@conf.theta.eu.org](xmpp:whatsxmpp@conf.theta.eu.org?join) if you're interested in using or running the bridge.

**Comment by Echolon** [comment]
@eeeeeta Thanks for your conrtibution. I added it in 1089710 with some minor changes if you agree.

**Comment by goffi-contrib** [comment]
@pmaziere I've just updated the theme on https://www.goffi.org (not yet on https://www.libervia.org, hopefully I'll do it soon, and I won't have time to update https://salut-a-toi.org before a little while), and I have a progress note close to be published, I still have to do 2 screenshots and check it, I think it will be online tomorrow. Sorry for the last minute notification, but you know, family life and all :). Thanks!

**Comment by goffi-contrib** [comment]
I have also just published a new progress note: https://www.goffi.org/b/n9nzn3ySF7Lv4pYBpTqYtP/progress-note

**Comment by pmaziere** [comment]
nicknames in the Thanks section should be reviewed: I may be wrong, but they do not seem to be set according to github nicknames as it is usually done

**Comment by Echolon** [comment]
Thanks @ge0rg 09a5678

**Comment by Echolon** [comment]
@erszcz 
about this sentence:
> support for Proxy Protocol which safely transport connection information

@ge0rg said: 
> This is in Mongoose and doesn't read right

Can you both align on this? I personally lag the knowledge to discuss or evaluate.

File: content/posts/newsletter/2020-08-06.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+### Servers
+
+[MongooseIM 3.7.1 has been released!](https://github.com/esl/MongooseIM/releases/tag/3.7.1)
+This one is built on top of the 3.7.0 May release, which introduced support for channel binding to prevent [replay attacks](https://en.wikipedia.org/wiki/Replay_attack)
+for all methods of [SCRAM](https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism "Salted Challenge Response Authentication Mechanism"),  a family of modern, password-based challenge–response authentication mechanisms providing authentication of a user to a server. This previous version also provided a range of new SCRAM authentication methods based on different flavors of the [SHA](https://en.wikipedia.org/wiki/Secure_Hash_Algorithms "Secure Hash Algorithms") cryptographic hash functions, the ability to retract messages as specified by [XEP-0424](https://xmpp.org/extensions/xep-0424.html), and support for [Proxy Protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) which safely transport connection information such as a client's address across multiple layers of NAT or TCP proxies. Version 3.7.1 improves on that by speeding up all the SCRAM methods significantly!
```
Comment (erszcz) [inline content/posts/newsletter/2020-08-06.md:64]:
```suggestion
for all methods of [SCRAM](https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism "Salted Challenge Response Authentication Mechanism"),  a family of modern, password-based challenge–response authentication mechanisms providing authentication of a user to a server. This previous version also provided a range of new SCRAM authentication methods based on different flavors of the [SHA](https://en.wikipedia.org/wiki/Secure_Hash_Algorithms "Secure Hash Algorithms") cryptographic hash functions, the ability to retract messages as specified by [XEP-0424](https://xmpp.org/extensions/xep-0424.html), and support for [Proxy Protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) which safely transports connection information such as a client's address across multiple layers of NAT or TCP proxies. Version 3.7.1 improves on that by speeding up all the SCRAM methods significantly!
```

Comment (erszcz) [reply content/posts/newsletter/2020-08-06.md:64]: This just changes the verb form which was introduced after my commit: "[Proxy protocol] which safely transport" -> "which safely transports".

Proxy Protocol was introduced in 3.7.0, which we missed the May newsletter with, hence a highlight of 3.7.0 features with the 3.7.1 announcement. Here are the 3.7.0 release notes: https://github.com/esl/MongooseIM/releases/tag/3.7.0

**Comment by erszcz** [comment]
@Echolon @ge0rg I've added a suggestion to fix the grammar - it should be fine now. Thanks for flagging this :+1:

File: content/posts/newsletter/2020-08-06.md (hunk 1 above)
Comment (Echolon) [reply content/posts/newsletter/2020-08-06.md]: Thanks! 6387ddd1f43a2de761fcee83dabffe8b28af93a5

**Comment by Echolon** [comment]
## :hourglass:  **Editorial closing** 

### :rocket: Release is tomorrow

### :pray: Thanks to everyone of you!  :handshake:  **eta, erszcz, Ge0rG, Holger, kriztan, jerome-poisson, jonas', Licaon_Kter, pmaziere, vanitasvitae, wurstsalat, woj-tek, zash**

**Pull request merged**